### DISCUSSION TITLE
Home Assistant Notifications

### DISCUSSION THREAD

**Discussion by jib-au** [djdd87/SynoAI#208] [General]
Firstly, thanks for developing SynoAI. 

My goal is to have SynoAi to push all camera notifications from Home Assistant (Hass) only. Pushbullet works fine, but its another app, and I am trying to streamline everything. 

I am trying to troubleshoot one notifier. Just the garage camera. Once the Garage camera is working, I'll add the following webhooks for each camera. I can see the entity in hass, but the issue is the state is always set to 'idle" even when its sending notifications via pushbullet.  I read your guides, not sure what I am doing wrong. 

Here is my configuration.yaml

camera:
  - platform: push
    name: Motion Garage
    webhook_id: motion_garage
    timeout: 1
    buffer: 1

Here is the Notifiers section from the Appsettings.json

    "Notifiers": [
        {
            "Type": "Pushbullet",
            "ApiKey": "xxx"
        },
        {   "Type": "Webhook",
			"Url": "http://x.x.x.x:8123/api/webhook/motion_garage",
			"Cameras": [ "Garage" ]
		}
    ],

Thanks.

**Comment by jib-au** [comment]
Sorry, worked it out. Took a while for the entity to pull the snapshot from synoAI. Now that I have the snapshot, I have added the automations respective to each camera to send notifications via hass.io.

**Reply by JoeyJoe6** [reply]
What would your Appsettings.json file entry look like?. Im trying to complete the same but send it to Synology Chat app

**Reply by jib-au** [reply]
[SynoAI.zip](https://github.com/djdd87/SynoAI/files/13462890/SynoAI.zip)
see attached, I've also included the .yml files for hass if you only want your notifications sent via hass.
